src/java/org/apache/cassandra/io/sstable/format/SSTableReader.java

```diff
@@ -2377,8 +2378,9 @@ public void tidy()
                     obsoletion.commit();
 
                 // don't ideally want to dropPageCache for the file until all instances have been released
-                INativeLibrary.instance.trySkipCache(desc.fileFor(Component.DATA), 0, 0);
-                INativeLibrary.instance.trySkipCache(desc.fileFor(Component.PRIMARY_INDEX), 0, 0);
+                StorageProvider.instance.invalidateFileSystemCache(desc.fileFor(Component.DATA));
```

Comment:
It was and is wrong - you should rather use something like `desc.version.format.getCacheableComponents` and implement that method accordingly for BIG and BTI formats

Reply:
reported a ticket:
https://datastax.jira.com/browse/STAR-1357

good for now?